packaging/tests/tas-restapi/.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+registries:
+  maven-alfresco-internal:
+    type: maven-repository
+    url: https://artifacts.alfresco.com/nexus/content/groups/internal
+    username: ${{secrets.NEXUS_USERNAME}}
+    password: ${{secrets.NEXUS_PASSWORD}}
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "22:00"
+    timezone: Europe/London
+  open-pull-requests-limit: 99
+  registries:
+  - maven-alfresco-internal
+  ignore:
+    # httpclient 4.5.7 and above normalize URIs removing empty path params
+    # check https://alfresco.atlassian.net/browse/ACS-3186 for more details
+    - dependency-name: "org.apache.httpcomponents:httpclient-osgi"
+      versions: ">= 4.5.7"
```

Comment:
Should we remove this additional `dependabot.yml` file and merge it into the base one, by defining a new `package-ecosystem` with the `directory` appropriately pointing to the `tas-restapi` folder, so that we can enforce this ignore rule?

My assumption is that this additional `dependabot.yml` file serves no purpose/won't be taken into consideration, correct me if I'm wrong!

Reply:
You're right :+1:

Reply:
In fact this would have been really useful to find before I started. I spent a while investigating exactly this issue of empty path params and didn't solve it. Instead I came to the conclusion that the tests for empty path params were not valuable and so removed them.

As an example one test checked that a GET request to the v1 endpoint `/sites//containers` returns a 200 and an empty list. I don't even agree with the asserted behaviour here.